staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/errors.go

```diff
@@ -32,14 +32,6 @@ import (
 // Avoid emitting errors that look like valid HTML. Quotes are okay.
 var sanitizer = strings.NewReplacer(`&`, "&amp;", `<`, "&lt;", `>`, "&gt;")
 
-// BadGatewayError renders a simple bad gateway error.
-func BadGatewayError(w http.ResponseWriter, req *http.Request) {
```

Comment:
Hi, @tallclair I'm not really sure but `BadGatewayError` might still be useful for "proxy" servers to apiserver. E.g. if we have a dedicated "kube-aggragtor" and when apiserver is not responding somehow(due to network, or a high latency..) then should we return a "BadGatewayError"? 😀